CHARTER.md

```diff
@@ -0,0 +1,71 @@
+# Aggie Robotics Charter
+
+## Preamble
+
+**Whereas** the students of the College of Engineering at North Carolina Agricultural and Technical State University seek opportunities to innovate and create in pursuit of understanding,
+
+**Whereas** opportunities for hands on application of engineering are not readily available to underclassmen,
+
+**Whereas** the Department of Electrical and Computer Engineering seeks to encourage students to apply their knowledge and seek new technical skills,
+
+**We**, the undersigned, found on this day the Aggie Robotics Club.
+
+## Mission
+
+The mission of Aggie Robotics is to provide practical educational opportunities and resources, for free innovation in: Robotics, Electrical Engineering, and Computer Science.
+
+## Membership
+
+Membership in Aggie Robotics contains 3 levels: Apprentice Membership, Journeyman Membership, and Crackerjack Membership. All members will receive a `Member Badge`. Members can advance in level by `badging`. The `badging` process will be defined in the by-laws and published openly.
```

Comment:
After some thinking, version one is more appropriate. I can't imagine explaining "crackerjack" to Dr. Kelly or his peers. 
However i made a recent comment about the badge system. Instead of changing the member title, we can have a "non-official" naming system. Analogous to military titles.